docs/resources/sms_task.md

```diff
@@ -0,0 +1,162 @@
+  If omitted, the default project in the region will be used. Changing this parameter will create a new resource.
+
+<a name="target_server_disks_object"></a>
+The `target_server_disks` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the disk name, e.g. "/dev/sda".
+  Changing this parameter will create a new resource.
+
+* `size` - (Required, int, ForceNew) Specifies the volume size in bytes. Changing this parameter will create a new resource.
+
+* `device_type` - (Required, String, ForceNew) Specifies the disk type. The value can be **NORMAL** and **BOOT**.
+  Changing this parameter will create a new resource.
+
+* `disk_id` - (Optional, String, ForceNew) Specifies the disk index, e.g. "0".
+  Changing this parameter will create a new resource.
+
+* `used_size` - (Optional, int, ForceNew) Specifies the used space in bytes. Changing this parameter will create a new resource.
+
+* `physical_volumes` - (Optional, List, ForceNew) Specifies an array of physical volume informations.
+  The [object](#physical_volumes_object) is documented below. Changing this parameter will create a new resource.
+
+<a name="physical_volumes_object"></a>
+The `physical_volumes` block supports:
+
+* `name` - (Required, String, ForceNew) Specifies the volume name. In Windows, it indicates the drive letter,
+  and in Linux, it indicates the device ID, e.g. "/dev/sda1".
+  Changing this parameter will create a new resource.
+
+* `size` - (Required, int, ForceNew) Specifies the volume size in bytes. Changing this parameter will create a new resource.
+
+* `device_type` - (Required, String, ForceNew) Specifies the partition type. The value can be **NORMAL** and **OS**.
+  Changing this parameter will create a new resource.
+
+* `file_system` - (Required, String, ForceNew) Specifies the file system type, e.g. "ext4".
+  Changing this parameter will create a new resource.
+
+* `mount_point` - (Required, String, ForceNew) Specifies the mount point, e.g. "/".
+  Changing this parameter will create a new resource.
+
+* `index` - (Required, Int, ForceNew) Specifies the serial number of the volume.
+  Changing this parameter will create a new resource.
+
+* `used_size` - (Optional, int, ForceNew) Specifies the used space in bytes.
+  Changing this parameter will create a new resource.
+
+* `uuid` - (Optional, String, ForceNew) Specifies the GUID of the volume.
+  Changing this parameter will create a new resource.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID in UUID format.
+* `state` - The status of the migration task.
+* `enterprise_project_id` - The enterprise project id of the target server.
+* `target_server_name` - The name of the target server.
+* `migrate_speed` - The migration rate, in MB/s.
+
+## Timeouts
+
+This resource provides the following timeouts configuration options:
+
+* `create` - Default is 5 minute.
+* `update` - Default is 5 minute.
```

Comment:
seems update timeout does not take effect like create

Reply:
fixed